### ISSUE TITLE
Aria#alert and repeat-message

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#70288]
@jrieken as far as I can tell, a screen reader would not announce again if the same message is being put. With the trick of having a counter, we enforce reading a message again even if it was already read. Maybe a better fix would be to unset the message after a delay? Not sure if that would break things though...

_Originally posted by @bpasero in https://github.com/Microsoft/vscode/issues/65499#issuecomment-471554585_

**Comment by jrieken** [comment]
fyi @pawelurbanski

**Comment by pawelurbanski** [comment]
Aria alert is the way to trigger reading certain information by the screen reader, which it cannot do by itself. If it could, it would never announce counts of items.
Screen readers, for example, have the functionality to inform item position information, such as: 2 of 17 in a  list view and so on. I keep it turned off, for the item itself is more important.
Therefore I think that it is safe to switch count off for everyone.
If we ever need to inform the user about some count value, let it be the part of the string containing the message passed to the alert function.

**Comment by jrieken** [comment]
@pawelurbanski I have pushed a change to disable repeat for suggest messages

**Comment by pawelurbanski** [comment]
Great! With those changes, settings I’ve tested, and a few points in the wiki, VS Code will be in a good shape for good experience.
I’ll take some time to get more familiar with TypeScript to contribute more fixes, which are not so urgent.
I will also publish the NVDA add-on, and describe workarounds it is implementing, so they may be future things to fix in VS Code itself.

Time to get some coffee and start working...
